From 6ee762ff6775232b2e97c1644454737fe7f36ee8 Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@apache.org>
Date: Tue, 25 Oct 2016 09:59:12 -0700
Subject: [PATCH 1988/2748] HADOOP-13669. Addendum patch 2 for KMS Server
 should log exceptions before throwing.

(cherry picked from commit 09ef97dccbd4c11aaeedf7503d414f999435ad91)

Change-Id: If1549e6dd396c2eac76ef45251aed7623cd809fd
---
 .../hadoop-kms/dev-support/findbugsExcludeFile.xml |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-kms/dev-support/findbugsExcludeFile.xml b/hadoop-common-project/hadoop-kms/dev-support/findbugsExcludeFile.xml
index 78c4ca6..929936d 100644
--- a/hadoop-common-project/hadoop-kms/dev-support/findbugsExcludeFile.xml
+++ b/hadoop-common-project/hadoop-kms/dev-support/findbugsExcludeFile.xml
@@ -43,6 +43,6 @@
   -->
   <Match>
     <Class name="org.apache.hadoop.crypto.key.kms.server.KMS"/>
-    <Bug pattern="REC"/>
+    <Bug pattern="REC_CATCH_EXCEPTION"/>
   </Match>
 </FindBugsFilter>
-- 
1.7.9.5

